tests/data/setup.py

```diff
@@ -14,6 +14,9 @@
 requirements = [
     'click>=5.0.0',
     'mock>=1.3.0',
+    'invenio-db[versioning]>=1',
+    'invenio-db[versioning,mysql]>=1',
```

Comment:
pip compile needs a requirements.txt file as input, the step from setup.py->requirements.txt is done by requirements-builder and that step is the one that's failing:

```
08:36 AM ~/Work/inveniosoftware/invenio-workflows<venv:invenio-workflows>  (starting-point-invenio-3|…7) 
dcaro@pcrcssis001$ grep invenio-db setup.py 
        'invenio-db[postgresql,versioning]>=1.0.0a9',
        'invenio-db[mysql,versioning]>=1.0.0a9',
        'invenio-db[versioning]>=1.0.0a9',
    'invenio-db[versioning]>=1.0.0a9',
08:36 AM ~/Work/inveniosoftware/invenio-workflows<venv:invenio-workflows>  (starting-point-invenio-3|…7) 
dcaro@pcrcssis001$ requirements-builder -e mysql --level=min setup.py
blinker==1.4
flask-celeryext==0.1.0
Flask-CLI==0.2.1
invenio-db==1.0.0a9
invenio-files-rest==1.0.0a3
invenio-records-files==1.0.0a5
```